default.conf

```diff
@@ -34,12 +45,30 @@ server {
     # /home
     # redirect to /app/assets/index.html
     location = /home {
-        add_header Cache-Control "no-cache";
+        if ($is_mobile = "1") {
+            return 301 $scheme://$http_host/mobilehome;
+        }
+
         # don't cache so the most uptodate js files are requested
+        add_header Cache-Control "no-cache";
         root /app/assets;
         try_files /index.html =404;
     }
 
+    # /mobilehome
+    # redirect to /app/assets/index.html
+    location = /mobilehome {
+        if ($is_mobile = "0") {
+            return 301 $scheme://$http_host/home;
+        }
+
+        # don't cache so the most uptodate js files are requested
+        add_header Cache-Control "no-cache";
+
+        root /app/assets;
+        try_files /mobile/index.html =404;
```

Comment:
yeah, so for a PWA app ... they should be served up by the `api_sails/views/mobile_pwa.ejs` file.  Not this `/mobile/index.html`.

Reply:
Okay, thanks!